docs/install/windows.rst

```diff
@@ -42,6 +42,8 @@ Uninstall
 Upgrade
 -------
 
+.. warning:: Make sure that Kolibri is not running before upgrading.
+
```

Comment:
If this is a Windows issue, I think we should refer to an issue report or Community Forum post where the reasoning for this warning is elaborated. Or just elaborate it here.

Reply:
Warnings like this are not at all unusual on Windows, I often restart the system before installing or upgrading anything, as a preventive measure :wink: